tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -14,6 +14,9 @@ spec:
   - name: region
     default: "us-west-2"
     description: The region where the cluster is in.
+  - name: kubernetes-version
+    default: "1.21"
```

Comment:
let's use EKS default version as default. 1.21 is not being supported anymore at this point.

Reply:
Did you mean to add a version here? 1.24?

Reply:
Default version is the version EKS uses when you attempt to create a EKS cluster without mentioning a version, i think it's 1.28 now.

Reply:
changed the default to 1.28